apps/docs/guide/introduction/installing.md

```diff
@@ -17,6 +17,8 @@ If not using Docker Desktop, Docker RISC-V support can be installed by running t
 docker run --privileged --rm tonistiigi/binfmt:riscv
 ```
 
+Sunodo also uses Anvil commands. To install Anvil, you need to follow the instructions [here](https://book.getfoundry.sh/getting-started/installation#using-foundryup)
+
```

Comment:
Also, `anvil` is a development dependency, maybe we should have it on a different section or document.

Reply:
Done